extract.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import os
+import shutil
+import subprocess
+import sys
+from contextlib import suppress
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+__VERSION__ = '0.1'
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='FACT extractor CLI')
+    parser.add_argument('-v', '--version', action='version', version=__VERSION__)
```

Comment:
It is more convenient to output "PROGRAM_NAME - VERSION_NUMBER". All our other tools do it this way.

Reply:
It would be nice to get information on the container version as well.